### ISSUE TITLE
[rule: spaced-comment] shebang errors if followed by import

### ISSUE THREAD

**Issue by t128** [eslint/eslint#3296]
errors:
```javascript
#!/usr/bin/env babel-node

import {spawn} from 'foobar';
```

works w/o errors:
```javascript
#!/usr/bin/env babel-node

import {spawn as spawn} from 'foobar';
```

```sh
$ eslint -v
v1.0.0
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by nzakas** [comment]
Per the last comment, we still need the following to triage your issue:

1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

**Comment by t128** [comment]
oh, sorry

```yaml
 extends: eslint-config-airbnb
 rules:
  indent:
    - 2
    - tab
    -
      SwitchCase: 1
  no-warning-comments:
    - 1
    -
      location: anywhere
```
[eslint-config-airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/.eslintrc)

```
scripts/watch
  1:1  error  Expected exception block, space or tab after // in comment  spaced-comment

✖ 1 problem (1 error, 0 warnings)
```

**Comment by mysticatea** [comment]
Umm, I cannot reproduce that with minimal set.

## .eslintrc

```
ecmaFeatures:
    modules: true
rules:
    spaced-comment: 2
```

## test1.js

```js
#!/usr/bin/env babel-node

import {spawn} from 'foobar';
```

## test2.js

```js
#!/usr/bin/env babel-node

import {spawn as spawn} from 'foobar';
```

## Actual

```
> eslint -v
v1.0.0
> eslint test1.js
> eslint test2.js
>
```

No errors.

**Comment by mysticatea** [comment]
Reproduced with `babel-eslint`.

```
parser: "babel-eslint"
rules:
    spaced-comment: 2
```

```
> eslint test1.js

test.js
  1:1  error  Expected space or tab after // in comment  spaced-comment

✖ 1 problem (1 error, 0 warnings)
```

**Comment by t128** [comment]
so it's a babel problem?

**Comment by olalonde** [comment]
Having same issue here. For some strange reason, it only gives me this error on some files and others not.

**Comment by olalonde** [comment]
So I found out that:

```javascript
#!/usr/bin/env node

import { BlockWorker } from '../Workers';
import config from '../config';
// ...
```

gives a spaced-comment error, while this does not:

```javascript
#!/usr/bin/env node

import config from '../config';
import { BlockWorker } from '../Workers';
// ...
```

**Comment by nzakas** [comment]
@hzoo possible babel-eslint problem

**Comment by olalonde** [comment]
Created an issue on babel-eslint (https://github.com/eslint/eslint/issues/3296).

**Comment by nzakas** [comment]
I believe it's a babel-eslint problem, so closing.

**Comment by hzoo** [comment]
Seems like the `LineComment` event is fired in `spaced-comment` for babel-eslint but not in the normal parser - I'm trying to figure out why..

It looks like in https://github.com/eslint/eslint/blob/c694db4740d3f8e3d9f908a4e7321c398e924225/lib/eslint.js#L804-L810 all the shebang comments are removed which seems to be the case in both cases since this happens after parse?
